apps/tailwind-components/pages/input/Decimal.story.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <InputTestContainer
+    show-placeholder
+    show-state
+    v-slot="{ placeholder, valid, invalid, disabled }"
+  >
+    <InputDecimal
```

Comment:
Maybe it would be good to add a label here and link it with the input (see below)? This gives context for the user and what they might expect.

```vue
<label id="input-decimal-title">Enter a decimal</label>
<InputDecimal describedBy="input-decimal-title" ..../>
```

Reply:
I would rather that we add Dcimal to the FormFields story. There you see how it all works together with a label.
(and then you could even consider to delete these input-only demos)